Lab 2.ipynb

```diff
@@ -0,0 +1,644 @@
eDVOyJiIiIhkt3xM2rQ0qIiIiGS+fu0dVWRMREZGskm/do6qsZZC9ffDy7ZeESCYys8OBa4GW7n5upK098HtgM/AO4emIJgFbgJfcfV4w0YrktnxZe1SVNRHJG2Y2w8w2mtmqmPaBZva2ma01swl72oe7r3P3MTHNPwCecffRQGfgx8Dd7j4O+FljPgcR+aZ8qLKpsiYi+WQm4QrYg7UNZlYATAP6E15hZYmZzQcKgKkx9x/t7hvj7HcZcK2ZlQEPAc8AEyNzR7Zu7CchIvXl8jltqqyJSN5w90XApzHNvYC1kYrZTmA2MNjdV7r7oJi/eIkawChgorv3A053943ufikwgXDXaFxmNtbMys2sfNOmTfv/BEUEyL3Ro0rWRCTfFQEfRV0PRdriMrPWZvZHoLuZ/TrS/Bzwi0h7lZmVmNl0whW8mxPty92nu3upu5e2bdt2v5+IiPxTLnWPqhtURPKdxWnzRBu7+xbg4pi2VcC5MZuO3f/QRGR/5cIgBCVrIpLvQsChUdeLgY8DikVEUiDbz2dTN6iI5LslQEcz62BmzYBhhKfeEJEclI3do0rWRCRvmNksYDFwhJmFzGyMu+8CxgPPA5XAXHdfHWScIpJ62ZSwqRtURPKGuw9P0L4AWJDmcEQkYLUJW6Z3jSpZExERkbyW6ee0qRtUREREJCITu0eVrImIiIhEybRBCErWREREROLIlIRNyZqIiIhIAplQZdMAgyyytw9LJp4UKSIikguCHISgypqIiIhIEtJdaVOyJiIiIpKkdHaPKlkTERER2UfpSNiUrImIiIjsh1RX2ZSsiYiIiGSwQEaDmtlA4E6gALjX3W+Muf0A4EGgJ7AFKHP3qnTHmW00WlQk9czsJ8DpwMHANHd/wcw6AZcDbYCXgIXAJMLHr5fcfV5A4YpIDkh7Zc3MCoBpwI+BzsBwM+scs9kYYKu7fx+4Hfiv9EYpIrnIzGaY2UYzWxXTPtDM3jaztWY2YU/7cPc/u/vPgZFAWaSt0t0vBoYCpYSPb3e7+zjgZ6l4LiKSP4KorPUC1rr7OgAzmw0MBtZEbTOY8K9SgHnA783M3N3TGWiuUeVNhJnA7wlX7oFv/IDsD4SAJWY2n3Dlf2rM/Ue7+8bI5d9E7le7nzOBCZH9vwhMjLS1TskzEZG8EUSyVgR8FHU9BPROtI277zKzbYQPeJujNzKzscDYyNUvPvivQW8nEUeb2P1loUZ9Dpb++mW2vwfZHj9k/3M4IpmN3X2RmZXENMf9AenuU4FBsfswMwNuBJ5196VR+54PzDezZ9z9UeDSSCL4RKJ4Yo9hZpbMMQyy9/3LxrgVc3pkY8wQiTvJ79HDGrphEMmaxWmLrZg1ZBvcfTowfZ+CMCt399J9uW+myPbnoPiDl+3PwczKG2E3DfkBGe0y4BSgpZl9393/aGYnAmcDBwALIgnhNUBz4OZEO9qfYxhk7/uXjXEr5vTIxpgh9XEHkayFgEOjrhcDHyfYJmRmTYCWwKfpCU9E8kyDfhzW3eB+F3BXTNurwKsxm45FRKQRBDF1xxKgo5l1MLNmwDBgfsw284ERkcvnAi/rfDURSZGG/IAUEQlM2pM1d98FjAeeByqBue6+2syuj5yMC3Af0NrM1gJXET5pt7Htc9dDBsn256D4g5ftz6Ex4m/ID8hMla3vXzbGrZjTIxtjhhTHbSpYiUi+MLNZwImETwb+BJjo7veZ2WnAHYRHgM5w9xuCi1JE5JuUrImIiIhkMC03JSIiIpLB8jJZS2a28kxjZoea2StmVmlmq83s8qBj2hdmVmBmy8zs6aBj2RdmdpCZzTOztyLvxQ+DjikZZnZl5POzysxmmVlh0DHtTbzVB8zsX83sr2b2buTfVkHGmE7ZchzLtvct0TE2k2MGMLNCM3vDzN6MxH1dpL2Dmf09EvecyHmZGSX2+yDTYzazKjNbaWbLa6cPSvXnI++SNWvYcleZbBfwS3fvBPQhPPFmNsVf63LCA0yy1Z3Ac+5+JHA0WfRczKwI+AVQ6u5dCZ+nNSzYqBpkJjAwpm0C4bU3OxJekzNjk5bGlGXHsZlk1/uW6BibyTEDfA30c/ejgWOAgWbWh/ByjbdH4t5KeDnHTBP7fZANMZ/k7sdEza2W0s9H3iVrRM1W7u47gdrlrrKCu2+onTXd3f9B+ANeFGxUyTGzYsILYd8bdCz7wswOBE4gPGoZd9/p7p8FG1XSmgD/EpnH8NtkwVQV7r6I+vMtDgYeiFx+APhJWoMKTtYcx7LtfdvDMTZjYwbwsC8iV5tG/hzoR3jZRsjAuGO/DyIrhGR0zAmk9PORj8lavNnKsyrZqRWZJb078PdgI0naHcC/AzVBB7KPDgc2AfdHSvf3mlnzoINqKHdfD9wCfAhsALa5+wvBRrXP/p+7b4DwlyxwcMDxpEu2H8ey4n2LOcZmfMyR7sTlwEbgr8B7wGeRKbMgMz8nsd8Hrcn8mB14wcwqLLxkHKT485GPyVpSs5VnKjP7DvA4cIW7fx50PA1lZoOAje5eEXQs+6EJ0AP4g7t3B74k87pEEoqcSzEY6AAcAjQ3s58GG5UkKSeOY5ksG4+x7r7b3Y8hPLFzL6BTvM3SG1ViCb4PsuGzfZy79yB8GsKlZnZCqh8wH5O1rJ+t3MyaEj6IPOLuCReJzlDHAWeaWRXhrpt+ZvZwsCElLQSE3L22ojmPcPKWLU4B3nf3Te5eTXih8R8FHNO++sTM2gFE/t0YcDzpku3HsYx+3xIcYzM65miR0zJeJXzO3UGR0x0g8z4n9b4PCFfaMjlm3P3jyL8bgScJJ8Yp/XzkY7KWzbOV1/bn3wdUuvttQceTLHf/tbsXu3sJ4df+ZXfPqqqOu/8f8JGZHRFpOhlYE2BIyfoQ6GNm3458nk4miwZIxIhemm4E8JcAY0mnrD6OkcHv2x6OsRkbM4CZtTWzgyKX/4Xwj7JK4BXCyzZChsWd4PvgfDI4ZjNrbmYtai8DA4BVpPrz4e5
```

Comment:
I think you need to adjust the widths on your plots, the peak shouldn't be so wide. What I did was set the width to x[1]-x[0]. The y-axis label on the bottom log plot is overlapping the other plot.